modules/upgrade/pages/proxy-uyuni.adoc

```diff
@@ -23,16 +23,16 @@ opensuse_leap15_2-uyuni-client \
 uyuni-proxy-stable-leap-152
 ----
 . Fully synchronize all channels with [command]``spacewalk-repo-sync``.
-    In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
+  In case of already defined repository URLs, continue with <<uyuni-202007-channeldupes>>.
 
 
 
 [[uyuni-202007-channeldupes]]
-== Uyuni 2020.07: Troubleshooting with Duplicated Repository URLs
+== Troubleshooting with Duplicated Repository URLs
 
 // https://github.com/SUSE/spacewalk/issues/12008
 
-If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} 2020.07 with [command]``spacewalk-common-channels``, you will see an error such as:
+If you want to add two or more repositories with the same URL (client tools for openSUSE Leap 15.0, 15.1, and 15.2) in {productname} with [command]``spacewalk-common-channels``, you will see an error such as:
```

Comment:
I see a small problem about this. the Leap channels on SUSE Manager should NOT be used from `spacewalk-common-channels` under any circumstances.

And right now the only "support" OS that uses `spacewalk-common-channels` on SUSE Manager is Debian, and AFAIK it's not affected by this.

Should we maybe have this for Uyuni only?

Reply:
ok, thanks for the background info.  I think thus far we publish this in the Uyuni context only (but we should be prepared).  I think I must read again what we say about using Leap with suma.

Reply:
in the client config, Lana put in nice ifdefs.  I think everything is fine and I can merge this one.